docs/en/index.md

```diff
@@ -124,3 +124,36 @@ You can disable the native driver by this option (and the emulated will take con
 redis:
 	session: {native: off}
 ```
+
+## Key prefixes
+
+When you use two instances of one application with one Redis server, it is possible, that your data can be overwritten by second application instance.
+
+To avoid this problem you can define key prefixes in configuration, for example:
+
+Instance 1
+```yml
+redis:
+    keyPrefix: "instance1_"
```

Comment:
Also, the `_` is added automatically in the classes. The resulting key would be `instance1__Nette:something`